README.md

```diff
@@ -15,8 +15,6 @@ To-be-renamed project repository for the HealthMap/GDO open epidemiology collabo
 
 ![Ingestion functions AWS SAM deploy](https://github.com/open-covid-data/healthmap-gdo-temp/workflows/Ingestion%20functions%20AWS%20SAM%20deploy/badge.svg)
 
-![current Docker status badge](https://github.com/open-covid-data/healthmap-gdo-temp/workflows/Docker%20compose%20up/badge.svg)
```

Comment:
Ah, I didn't add one of these for the integration tests yml (https://github.com/open-covid-data/healthmap-gdo-temp/blob/master/.github/workflows/integration-tests.yml). Could you add one in while you're here?

Reply:
Yep, done!